src/pint/observatory/topo_obs.py

```diff
@@ -136,7 +153,11 @@ def __init__(
         tempo_code=None,
         itoa_code=None,
         aliases=None,
+        location=None,
         itrf_xyz=None,
+        lat=None,
+        lon=None,
+        height=None,
```

Comment:
Er, don't the docs call fot `alt` rather than `height`?

Reply:
I switched back to `height` which is the `EarthLocation` standard after discussion. I just forgot to update the docs.  That is now done.